compact/central_beampipe.xml

```diff
@@ -62,7 +62,8 @@
           <zplane z="BeampipeDownstreamStraightLength + 0.5 * BeampipeOD * tan(abs(CrossingAngle))" OD="BeampipeOD"/>
           <zplane z="1750.00 * mm" OD=" 92.06 * mm"/>
           <zplane z="4455.80 * mm" OD="257.92 * mm"/>
-          <zplane z="5000.00 * mm" OD="340.60 * mm"/>
+          <!-- removed due to vacuum overlap with far forward electron beampipe -->
+          <!--zplane z="5000.00 * mm" OD="340.60 * mm"/-->
```

Comment:
Don't we have a gap in the beampipe now? How long is the gap? Can we close this gap?

Reply:
The last segment of the beampipe has been removed for now (from 4455.8 mm to 5000.0 mm - 544.2 mm). This part is the outer shell, which houses both the electron and ion beampipes inside of the hcal. The electron beampipe that is being added matches the end of the central section. The ion beampipe is not implemented in main as far as I see.
The reason for this is, that during the placement of this beampipe vacuum is inserted on the inside. This vacuum then overlaps with the electron beampipe that is being added.

Reply:
From what I can see the ion beampipe vacuum isn't continuous in the simulation to begin with, surely this can't be the case? I can't see anything defined between the magnet volumes. Is the shape of the beampipe joining the central pipe and B0 well defined?

Reply:
Unfortunately I have no information regarding the ion beampipe. From what I see in both main and this branch, the far forward ion beamline is not implemented.
As for the shape of the added segment and the central pipe, I am confident that they match properly.
In this case, the electron beampipe, which is implemented in the central part, ends at 4484.25 mm, after which length the vacuum extends up to 5000 mm. In order to match the two electron beampipe parts, this vacuum needs to be somehow removed.

Reply:
@ajentsch can you comment on whether the ion beamline vacuum is currently continuous or if it has been left as air between the magnets for a particular reason?

Reply:
Hi all,

I would just like to ask, if there is anything that needs to be changed in this request on my side. The implemented beampipe will be needed in order to continue the background simulation effort.